### PR TITLE
Blog index redirected in all sections to query-index file

### LINKED ISSUE #123: FR: Blog section update pointing to blog index

### PR DESCRIPTION
**Pull request by SantiagoHomps-NC**
Fix #123

Test URLs:
- Before: https://main--vg-roadchoice-com--hlxsites.hlx.page/
- After: https://123-blog-index--vg-roadchoice-com--hlxsites.hlx.page/

- Before: https://main--vg-roadchoice-com--hlxsites.hlx.page/blog/
- After: https://123-blog-index--vg-roadchoice-com--hlxsites.hlx.page/blog/

### CODE REVIEW

**Review by manuel-vara — CHANGES_REQUESTED** [review]
I see some errors in the console:
image

File: blocks/blog-results/blog-results.js [hunk 1]
```diff
@@ -2,7 +2,6 @@ import {
   createElement,
   getTextLabel,
   getJsonFromUrl,
-  convertDateExcel,
```
Comment (jonatan-lledo-netcentric) [inline blocks/blog-results/blog-results.js:5]: `convertDateExcel` function is still needed?

**Review by jonatan-lledo-netcentric — CHANGES_REQUESTED** [review]
apart from that, it's ok to me

File: blocks/blog-results/blog-results.js (hunk 1 above)
Comment (SantiagoHomps-NC) [reply blocks/blog-results/blog-results.js:5]: no, I missed deleting it... strange the linter didnt remind me

Comment (SantiagoHomps-NC) [reply blocks/blog-results/blog-results.js:5]: aha! you are looking to the deleted part

**Review by jonatan-lledo-netcentric — APPROVED** [review]

**Review by manuel-vara — APPROVED** [review]

**Pull request merged**